### DISCUSSION TITLE
Author

### DISCUSSION THREAD

**Discussion by jmalinie** [GetPublii/Publii#1414] [General]
Hi,
i m looking for a solution for ;

* Authors will be able to login with their e-mail and be able to publish on my blog. 
* They will not be able to see & manage the site settings and other authors posts like on blogger and wordpress.

is publii doing it already ? if so how ? 

Thanks

**Comment by dziudek** [comment]
Hi,

Unfortunately Publii has no support for such features - it is an app dedicated for single-user usage. In theory you can share your sites via dropbox and then other people can also edit them, but it is not designed currently for collaboration.

**Comment by candidexmedia** [comment]
Besides what Tomasz has already explained, I would suggest rethinking the editorial process if you're keen on using Publii. You could assign one or two editors who are in charge of pasting in the work of other writers into Publii. 

Otherwise, [Ghost](https://ghost.org/help/managing-your-team/#:~:text=To%20invite%20new%20users%20to,base%20user%20level%20in%20Ghost.) might be better suited for your use case. You can self-host Ghost affordably using 1-click deploy services like Railway and Pikapods, or the host of your choice.
